src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -1234,6 +1235,8 @@ export interface ExtHostDocumentsAndEditorsShape {
 export interface ExtHostTreeViewsShape {
 	// {{SQL CARBON EDIT}}
 	$getChildren(treeViewId: string, treeItemHandle?: string): Promise<sqlITreeItem[]>;
+	// {{SQL CARBON EDIT}}
```

Comment:
Why are we needing to do a SQL CARBON EDIT here if you're just cherry picking commits? This looks like what VS Code has

Reply:
I added the `?` operator, because it was throwing an error for not implementing the drop method in `ExtHostModelViewTreeViewsShape` https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/api/common/sqlExtHost.protocol.ts#L741

Reply:
Not sure if it's best to implement the drop method in the sql files as well

Reply:
That would be the preferred approach - we should avoid changing VS files whenever we can

Reply:
And in the future - if you add a SQL EDIT tag please always include a brief description in the comment about why that change is needed for other people looking at it in the future

Reply:
Sounds good! I'll make those changes